ChangeLog

```diff
@@ -4,7 +4,27 @@ Pylint's ChangeLog
 
 What's New in Pylint 2.8.0?
 ===========================
+..
+  Put new features here
 
+What's New in Pylint 2.7.2?
+===========================
+..
+  Put bug fixes that will be cherry-picked to latest major version here
+
+* Fix False Positive on `Enum.__members__.items()`, `Enum.__members__.values`, and `Enum.__members__.keys`
+  
```

Comment:
```suggestion

```
There are two trailing-whitespaces here which cause the pre-commit check to fail.
If you want, you could also run pre-commit locally with:
```bash
pip install pre-commit
pre-commit install
```
That will run the pre-commit checks on every commit.